changelogs/fragments/add_type_checking_to_role_init.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
```

This is probably not a bug fix:
```suggestion
minor_changes:
```